app/views/admin/_header.html.erb

```diff
@@ -58,7 +60,9 @@
       </ul>
     </li>
   <% end %>
-  <li><%= link_to ts("Tag Wrangling", key: "header"), tag_wranglings_path %></li>
+  <% if policy(WranglingGuideline).new? %>
```

Comment:
We're not actually restricting Tag Wrangling in this PR, so let's leave the Tag Wrangling link as-is.